src/tcfv2/stub.js

```diff
@@ -1,297 +1,91 @@
-						}
-					},
-				});
-		},
-		function (t, e, n) {
-			var r = n(0),
-				o = n(6),
-				i = n(10),
-				c = n(11),
-				u = Object.defineProperty;
-			e.f = r
-				? u
-				: function (t, e, n) {
-						if ((i(t), (e = c(e, !0)), i(n), o))
-							try {
-								return u(t, e, n);
-							} catch (t) {}
-						if ('get' in n || 'set' in n)
-							throw TypeError('Accessors not supported');
-						return 'value' in n && (t[e] = n.value), t;
-				  };
-		},
-		function (t, e, n) {
-			var r = n(0),
-				o = n(2),
-				i = n(7);
-			t.exports =
-				!r &&
-				!o(function () {
-					return (
-						7 !=
-						Object.defineProperty(i('div'), 'a', {
-							get: function () {
-								return 7;
+			n.addEventListener(
+				'message',
+				function (e) {
+					var t = 'string' == typeof e.data,
+						a = {};
+					try {
+						a = t ? JSON.parse(e.data) : e.data;
+					} catch (e) {}
+					var n = a.__tcfapiCall;
+					n &&
+						window.__tcfapi(
+							n.command,
+							n.version,
+							function (a, r) {
+								var i = {
+									__tcfapiReturn: {
+										returnValue: a,
+										success: r,
+										callId: n.callId,
+									},
+								};
+								t && (i = JSON.stringify(i)),
+									e.source.postMessage(i, '*');
 							},
-						}).a
-					);
-				});
-		},
-		function (t, e, n) {
-			var r = n(8),
-				o = n(1),
-				i = r.document,
-				c = o(i) && o(i.createElement);
-			t.exports = function (t) {
-				return c ? i.createElement(t) : {};
-			};
-		},
-		function (t, e, n) {
-			(function (e) {
-				var n = function (t) {
-					return t && t.Math == Math && t;
-				};
-				t.exports =
-					n('object' == typeof globalThis && globalThis) ||
-					n('object' == typeof window && window) ||
-					n('object' == typeof self && self) ||
-					n('object' == typeof e && e) ||
-					Function('return this')();
-			}.call(this, n(9)));
-		},
-		function (t, e) {
-			var n;
-			n = (function () {
-				return this;
-			})();
-			try {
-				n = n || new Function('return this')();
-			} catch (t) {
-				'object' == typeof window && (n = window);
-			}
-			t.exports = n;
-		},
-		function (t, e, n) {
-			var r = n(1);
-			t.exports = function (t) {
-				if (!r(t)) throw TypeError(String(t) + ' is not an object');
-				return t;
-			};
-		},
-		function (t, e, n) {
-			var r = n(1);
-			t.exports = function (t, e) {
-				if (!r(t)) return t;
-				var n, o;
-				if (
-					e &&
-					'function' == typeof (n = t.toString) &&
-					!r((o = n.call(t)))
-				)
-					return o;
-				if ('function' == typeof (n = t.valueOf) && !r((o = n.call(t))))
-					return o;
-				if (
-					!e &&
-					'function' == typeof (n = t.toString) &&
-					!r((o = n.call(t)))
-				)
-					return o;
-				throw TypeError("Can't convert object to primitive value");
-			};
-		},
-	]);
+							n.parameter,
+						);
+				},
+				!1,
+			));
+	};
+	// call `e` directly. See https://git.io/JJxcM
+	// 'undefined' != typeof module ? (module.exports = e) : e();
+	e();
```

Comment:
@mxdvl calling `e` here puts `__tcfapi` on the `window` so no need to set a timeout in init!